include/ur_api.h

```diff
@@ -8423,6 +8423,25 @@ urKernelSuggestMaxCooperativeGroupCountExp(
     uint32_t *pGroupCountRet    ///< [out] pointer to maximum number of groups
 );
 
+///////////////////////////////////////////////////////////////////////////////
```

Comment:
@AllanZyne , need to fill and check this doc content

Reply:
Thanks, this is a new API. I've noticed this problem and filed an issue to track this.